genie-web/src/main/resources/db/migration/h2/V4_0_0__Genie_4.sql

```diff
@@ -65,3 +65,36 @@ ALTER TABLE `criteria`
   ADD COLUMN `version` VARCHAR(255) DEFAULT NULL;
 ALTER TABLE `criteria`
   ADD COLUMN `status` VARCHAR(255) DEFAULT NULL;
+
+CREATE TABLE `job_requested_environment_variables` (
```

Comment:
Seems we don't preserve orders of these. I suppose that's ok as long as we make it clear that setting
```
A=foo
...
A=bar
```
does not guarantee that the value of `A` will be `bar`

Reply:
Yes. Currently in memory these are stored in a Map so no order is applied. This is (currently) meant to only be the `variables passed in by user explicitly` + `those set by server (GENIE_*)`. I currently was not planning on storing any variables set by the setup scripts at runtime. If we decide that is the case I still suspect we'll only store the final variable state (e.g. what would be printed by an `env` command) so it should be one key to one value.